rafal_szyinski/main.cpp

```diff
@@ -43,7 +43,12 @@ void loop()
 			mode_button_state = reading;
 			if (mode_button_state == HIGH)
 			{ 
-					change_mode();
+				turn_off_color();
```

Comment:
Don't affect here on the circuit.
It would be better to move `turn_off_color()` before `switch` in the 56 line